### PR TITLE
Fix migrator

### PR DESCRIPTION
**Pull request by Art3miX**
Fixed the issue, and added a specific test to the error we were getting

### CODE REVIEW

File: contracts/external/dao-migrator/src/contract.rs [hunk 1]
```diff
@@ -88,18 +88,23 @@ fn execute_migration_v1_v2(
     }
 
     // List of code ids pairs we got and the migration msg of each one of them.
-    let proposal_pairs: Vec<CodeIdPair> = migration_params
+    let proposal_pairs: Vec<(String, CodeIdPair)> = migration_params
         .proposal_params
```
Comment (0xekez) [inline contracts/external/dao-migrator/src/contract.rs:92]: just a little thing, but could we add a check that this does not have duplicate `addr` values?

**Review by 0xekez — APPROVED** [review]
looks good! nice work on the alphabetical issue test. would be nice to deduplicate the proposal migration list, and then this is good to go.

**Review by JakeHartnell — APPROVED** [review]

File: contracts/external/dao-migrator/src/contract.rs (hunk 1 above)
Comment (Art3miX) [reply contracts/external/dao-migrator/src/contract.rs:102]:
```rust 
let mut uniq = HashSet::new();
if migration_params.proposal_params.iter().all(|(addr, _)| uniq.insert(addr)) {
    return Err(ContractError::DuplicateProposalParams);
}
```

Probably the nicest way I found to check for duplicates, not sure if HashSet is supported on the blockchain (it should in theory)

**Review by NoahSaso — APPROVED** [review]
woooo yay let's get this 🍞

File: contracts/external/dao-migrator/src/contract.rs (hunk 1 above)
Comment (0xekez) [reply contracts/external/dao-migrator/src/contract.rs:102]: looks great!

**Pull request merged**